chroma-manager/tests/framework/utils/selective_auto_pass.sh

```diff
@@ -51,6 +51,12 @@ check_for_autopass() {
         fi
     done
 
+    t="integration-tests-shared-storage-configuration-with-simulator"
+    if [[ $JOB_NAME == $t || $JOB_NAME == $t/* ]]; then
+        fake_test_pass "tests_skipped_because_agent_removed" "$WORKSPACE/test_reports/" ${BUILD_NUMBER}
+        exit 0
+    fi
+
```

Comment:
This doesn't really need to be here since it will never be reached with `chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/` gone also.

Reply:
Indeed. I'll fix in a follow-on